packages/components/Switch/src/Switch.styling.ts

```diff
@@ -0,0 +1,84 @@
+import { switchName, SwitchTokens, SwitchSlotProps, SwitchProps } from './Switch.types';
+import { UseStylingOptions, Theme, buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, layoutStyles, shadowStyles } from '@fluentui-react-native/tokens';
+import { defaultSwitchTokens } from './SwitchTokens';
+
+export const switchStates: (keyof SwitchTokens)[] = [
+  'toggleOn',
+  'toggleOff',
+  'before',
+  'after',
+  'above',
+  'hovered',
+  'focused',
+  'pressed',
+  'disabled',
+];
+
+export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, SwitchTokens> = {
+  tokens: [defaultSwitchTokens, switchName],
+  states: switchStates,
+  slotProps: {
+    root: buildProps(
+      (tokens: SwitchTokens) => ({
+        style: {
+          alignItems: 'center',
+          flexDirection: tokens.flexDirection,
+          alignSelf: 'flex-start',
+          minHeight: 28,
```

Comment:
Some of these seem like things that we would want to let clients customize, like minWidth, or the borderRadius of the thumb, or padding. Is that something you were thinking of changing later?

Reply:
Yeah at the moment I wasn't focusing on having it customizable. I just wanted the bones of it down.